corehq/apps/geospatial/views.py

```diff
@@ -435,3 +439,110 @@ def get_users_with_gps(request, domain):
     ]
 
     return json_response({'user_data': user_data})
+
+
+@method_decorator(toggles.GEOSPATIAL.required_decorator(), name="dispatch")
+class CasesReassignmentView(BaseDomainView):
+    urlname = "reassign_cases"
+    MAX_REASSIGNMENT_REQUEST_CASES = 100
+    SYNC_CASES_UPDATE_THRESHOLD = 1000
+    ASYNC_CASES_LIMIT = 5000
+
+    def post(self, request, domain, *args, **kwargs):
+        try:
+            request_data = json.loads(request.body)
+        except json.decoder.JSONDecodeError:
+            return HttpResponseBadRequest(
+                _('POST Body must be a valid json')
+            )
+
+        case_id_to_owner_id = request_data.get('case_id_to_owner_id', {})
+        include_related_cases = request_data.get('include_related_cases')
+        if len(case_id_to_owner_id) > self.MAX_REASSIGNMENT_REQUEST_CASES:
+            return HttpResponseBadRequest(
+                _("Maximum number of cases that can be reassigned is {limit}").format(
+                    limit=self.MAX_REASSIGNMENT_REQUEST_CASES
+                )
+            )
+
+        if include_related_cases:
+            request_cases_id = list(case_id_to_owner_id.keys())
+            parent_to_child_cases_id = self.get_child_cases(domain, request_cases_id)
```

Comment:
nit: Just want to remind about [this concept](https://www.baeldung.com/cs/clean-code-formatting#2-vertical-distance) for better flow to methods for readability. 

`get_child_cases` is referred here but is defined way lower. This makes it harder to follow code and in the PR, review.

Reply:
Addressed in 3b2c63bf899a98fd32f352bfc881c2a7f7077ed6